fibbingnode/misc/mininetlib/ipnet.py

```diff
@@ -423,9 +452,10 @@ def _add_node(self, n, props):
             nh = otherIntf(itf)
             if not nh:
                 continue  # Skip loopback and the likes
-            props[nh.node.name] = {
+            props[itf.name] = {
```

Comment:
Why is that change needed ? It prevents from accessing link information easily, e.g. db[a][b][ip] to know the ip address of a towards b.